packages/Webkul/Category/src/Database/Seeders/CategoryTableSeeder.php

```diff
@@ -41,7 +45,47 @@ public function run()
                 'meta_keywords'    => '',
                 'category_id'      => '1',
                 'locale'           => 'en',
+            ],
+            [
+                'name'             => 'Raíz',
+                'slug'             => 'root',
+                'description'      => 'Raíz',
+                'meta_title'       => '',
+                'meta_description' => '',
+                'meta_keywords'    => '',
+                'category_id'      => '1',
+                'locale'           => 'es',
+            ],
+            [
+                'name'             => 'Racine',
+                'slug'             => 'root',
+                'description'      => 'Racine',
+                'meta_title'       => '',
+                'meta_description' => '',
+                'meta_keywords'    => '',
+                'category_id'      => '1',
+                'locale'           => 'fr',
+            ],
+            [
+                'name'             => 'Wortel',
```

Comment:
The correct word is "Hoofdcategorie" wortel means carrot

Reply:
Mentioned changes have been done.